docs/DeveloperGuide.md

```diff
@@ -389,7 +398,7 @@ The following activity diagram summarizes what happens when a user enters a `fin
 <span style="font-size: large; color: #e46c0a">Design Considerations:</span>
 
 **Aspect: Allowing searching only through all fields**
-* As the hotel manager would usually appreciate the ability to search by other fields such as by Room, it is appropriate to make the `find` command to scan through all the fields of the guests.
+* As the hotel manager would usually appreciate the ability to search by other fields such as by `room`, it is appropriate to make the `find` command to scan through all the fields of the guests.
```

Comment:
Should we use "the user" here rather than "hotel manager", since we used "the user" as well in other areas?

Reply:
Yes!